src/components/CreateCoursePage/index.jsx

```diff
@@ -35,6 +35,13 @@ class CreateCoursePage extends React.Component {
     this.setStartedFetching();
   }
 
+  componentDidUpdate(prevProps) {
+    // check if errors on form submission and scroll to them
+    if (this.props.courseInfo.error && (!prevProps.courseInfo || !prevProps.courseInfo.error)) {
```

Comment:
This is checking that previous props did not have course info and error but current props have errors, correct?

Reply:
yes